src/document.c

```diff
@@ -366,9 +355,11 @@ void AddDocumentCtx_Free(RSAddDocumentCtx *aCtx) {
     aCtx->byteOffsets = NULL;
   }
 
-  if (aCtx->tokenizer) {
+  RS_LOG_ASSERT(aCtx->spec, "No indexer");
```

Comment:
Why added this assert?

Reply:
@OfirMos ^^

Reply:
Cause we are accessing aCtx->spec couple of lines later, and I want to catch a scenario when the sp isn't allocated because I has a bug like this as far as I can remember.